packages/core/src/estimate.ts

```diff
@@ -59,13 +57,14 @@ const estimateToEstimation = (estimate: Estimate): Estimation => ({
 });
 
 // Converts a known L1 error message to a more user-friendly one
-export const handleTezError = (err: Error): string => {
+export const handleTezError = (err: Error): string | undefined => {
   if (err.message.includes("subtraction_underflow")) {
     return "Insufficient balance, please make sure you have enough funds.";
   } else if (err.message.includes("contract.non_existing_contract")) {
     return "Contract does not exist, please check if the correct network is selected.";
   } else if (err.message.includes("staking_to_delegate_that_refuses_external_staking")) {
     return "The baker you are trying to stake to does not accept external staking.";
+  } else if (err.message.includes("empty_implicit_delegated_contract")) {
+    return "Emptying an implicit delegated account is not allowed.";
```

Comment:
nit: maybe something like 

`Emptying an implicit delegated account is not allowed. End delegation before ?trying? again.`

Reply:
Changed. Thanks